apollo-portal/src/main/resources/static/open/manage.html

```diff
@@ -61,24 +61,33 @@ <h5>{{'Open.Manage.CreateThirdApp' | translate }}
                             <tr>
                                 <th style="width: 10%">{{'Common.AppId' | translate }}</th>
                                 <th style="width: 15%">{{'Common.AppName' | translate }}</th>
-                                <th style="width: 15%">{{'Common.AppOwner' | translate }}</th>
+                                <th style="width: 10%">{{'Open.Manage.Consumer.AllowCreateApplication' | translate }}</th>
                                 <th style="width: 20%">{{'Common.Department' | translate }}</th>
-                                <th style="width: 20%">{{'Common.Email' | translate }}</th>
+                                <th style="width: 20%">{{'Common.AppOwner' | translate }}/{{'Common.Email' | translate }}</th>
                                 <th style="width: 20%">{{'Common.Operation' | translate}}</th>
                             </tr>
-                            <tr ng-repeat="app in consumerList" href="#">
-                                <td style="width: 10%">{{ app.appId }}</td>
-                                <td style="width: 15%">{{ app.name }}</td>
-                                <td style="width: 15%">{{ app.ownerName }}</td>
-                                <td style="width: 20%">{{ app.orgName + '(' + app.orgId + ')' }}</td>
-                                <td style="width: 20%">{{ app.ownerEmail }}</td>
+                            <tr ng-repeat="consumer in consumerList" href="#">
+                                <td style="width: 10%">{{ consumer.appId }}</td>
+                                <td style="width: 15%">{{ consumer.name }}</td>
+
+                                <td style="width: 15%" >
+                                    <div ng-if="consumer.allowCreateApplication">
+                                        <b style="color: red">{{'Open.Manage.Consumer.AllowCreateApplication.Yes' | translate}}</b>
```

Comment:
Is highlighting it in red necessary?

Reply:
removed red in style

![image](https://github.com/apolloconfig/apollo/assets/15523186/74709a89-6178-4c41-b438-ce61dd21f112)